src/domain/collaboration/callout/creation-dialog/CalloutTemplate/CalloutTemplateProps.ts

```diff
@@ -0,0 +1,5 @@
+import { CalloutSumaryProps } from '../../CalloutSummary';
```

Comment:
I believe this is not needed anymore.

Reply:
It's needed in the `CalloutCanvasTemplate` and `CalloutAspectTemplate`.